src/styles/block/container.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  position: relative;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.